src/base/TilesRendererBase.js

```diff
@@ -3,6 +3,7 @@ import { LRUCache } from '../utilities/LRUCache.js';
 import { PriorityQueue } from '../utilities/PriorityQueue.js';
 import { determineFrustumSet, toggleTiles, skipTraversal, markUsedSetLeaves, traverseSet } from './traverseFunctions.js';
 import { UNLOADED, LOADING, PARSING, LOADED, FAILED } from './constants.js';
+import { getTileContentType } from "../utilities/tileContentType";
```

Comment:
nit: single quotes (just updated the lint rules to catch this)

Reply:
Ah yes, thanks! Since prettier was released I started not paying attention to these sort of things and now I need a linter to tell me when I'm doing it wrong :)

Reply:
Yeah I didn't realize the linter wasn't catching this - I updated it a few days ago to catch quotes.